sql/src/main/java/io/crate/expression/scalar/ScalarFunctionModule.java

```diff
@@ -95,6 +96,7 @@ public void register(FunctionName qualifiedName, FunctionResolver functionResolv
 
     @Override
     protected void configure() {
+        PgTypeofFunction.register(this);
```

Comment:
minor: can we move this down to the group of pg system functions near line 170 for readability?